apps/web/app/(authenticated)/(app)/app/mobile-sidebar.tsx

```diff
@@ -59,6 +58,9 @@ export const MobileSideBar = ({ className }: Props) => {
           </div>
         </SheetContent>
       </Sheet>
+      <div className="absolute top-4 right-4">
+        <UserButton />
```

Comment:
This is the Clerk user button. can we not do what we did on desktop and shove a sign out button in the menu?